### PR TITLE
greensock animation

### PR DESCRIPTION
**Pull request by Caseyyu2**
v1 trail - chaining not working

### CODE REVIEW

**Comment by infiman** [comment]
@Caseyyu2 Is that platform (boilerplate) level issue or it's just your experiments with GSAP and you don't know for sure how it works yet?

**Comment by infiman** [comment]
BTW, this boilerplate is not complete yet and need to be updated with the latest universal features from https://github.com/infiman/swapi-fe

File: src/client/App/index.jsx [hunk 1]
```diff
@@ -1,26 +1,54 @@
...
-import './assets/index.scss';
+function createAnim() {
+  const box = document.getElementById('box');
+
+  const timelineM = new TimelineMax(box);
+  // TODO: why not chaining, why why why!
+  const tween1 = TweenMax.to(box, 1, { scale: 1.5, y: '+120', yoyo: true });
+  const tween2 = TweenMax.to(box, 1, { scale: 1, y: '-120', yoyo: true });
+  timelineM.repeat(-1);
+  timelineM.add([tween1, tween2], 0.5);
```
Comment (infiman) [inline src/client/App/index.jsx]: @Caseyyu2 for chaining you should use the last parameter of `.add(*, *, *, stagger)` method. See: tutorial https://greensock.com/position-parameter and reference in API Docs https://greensock.com/docs/TimelineLite/add()

What we have in the current example - you are adding both animations and telling them to tween with half a second delay. They both are running simultaneously but the second one has overridable priority (last in the array and the latest in the call stack), so it's overriding `y` and `scale` parameter from the first animation. Try to replace your animations with something like this to understand what I've meant:

```
const tween1 = TweenMax.to(box, 1, { y: '+120', yoyo: true });
const tween2 = TweenMax.to(box, 1, { scale: 1.5, yoyo: true });
```

**Review by infiman — COMMENTED** [review]
BTW, don't forget to reload the page every time you are updating code. React HMR is tricky with events and probably require additional handling for updates.

**Comment by infiman** [comment]
@Caseyyu2 what's the point of this PR?

**Comment by Caseyyu2** [comment]
@infiman was a convenience to fork and push. I think It makes more sense for me to download your code create my own repo, sorry for the pr noise. And thanks for the boiler plate 👍

**Comment by infiman** [comment]
@Caseyyu2 you still can work in your own fork just while creating PRs choose your own fork and not this repo unless you want to propose some changes to boilerplate.

**Pull request closed without merging**